frontend/admin/src/js/components/poolCandidate/ApplicationStatusForm/ApplicationStatusForm.tsx

```diff
@@ -0,0 +1,267 @@
+      notes: application.notes ?? "",
+      expiryDate: application.expiryDate
+        ? strToFormDate(application.expiryDate)
+        : undefined,
+    },
+  });
+  const { handleSubmit } = methods;
+
+  const handleFormSubmit: SubmitHandler<FormValues> = (values: FormValues) => {
+    onSubmit({
+      status: values.status,
+      notes: values.notes,
+      expiryDate: values.expiryDate,
+    });
+  };
+
+  return (
+    <div data-h2-width="base(100%)">
+      <Heading level="h2" data-h2-font-size="base(h5)">
+        {intl.formatMessage({
+          defaultMessage: "Application status",
+          id: "/s66sg",
+          description: "Title for admins to edit an applications status.",
+        })}
+      </Heading>
+      <div
+        data-h2-background-color="base(light.dt-gray)"
+        data-h2-radius="base(x1)"
+        data-h2-padding="base(x1, x1, x1, x1)"
+      >
+        <FormProvider {...methods}>
+          <form onSubmit={handleSubmit(handleFormSubmit)}>
+            <Heading
+              level="h3"
+              data-h2-font-size="base(h5)"
+              data-h2-margin="base(0, 0, x0.25, 0)"
+              data-h2-display="base(flex)"
+              data-h2-align-items="base(center)"
+              data-h2-gap="base(0, x0.25)"
+            >
+              <CalendarIcon
+                data-h2-width="base(0.75em)"
+                data-h2-height="base(0.75em)"
+                data-h2-margin="base(0, x0.25, 0, 0)"
+              />
+              <span>
+                {intl.formatMessage({
+                  defaultMessage: "Candidate status",
+                  id: "ETrCOq",
+                  description:
+                    "Title for admin editing a pool candidates status",
+                })}
+              </span>
+            </Heading>
+            <Select
+              label={intl.formatMessage({
+                defaultMessage: "Pool status",
+                id: "4idVCF",
+                description: "Label for the current applications pool status",
+              })}
+              required
+              rules={{ required: true }}
+              id="status"
+              name="status"
+              nullSelection={intl.formatMessage({
+                defaultMessage: "Select a status",
+                id: "VMhVyJ",
+                description: "Placeholder text for the pool status field",
+              })}
+              options={enumToOptions(PoolCandidateStatus).map(({ value }) => ({
+                value,
+                label: intl.formatMessage(getPoolCandidateStatus(value)),
+              }))}
+            />
+            <Input
+              id="expiryDate"
+              name="expiryDate"
+              label={intl.formatMessage({
+                defaultMessage: "Pool expiry date",
+                id: "W4O1Ip",
+                description:
+                  "Label displayed on the pool candidate application form expiry date field.",
+              })}
+              type="date"
```

Comment:
minor thing, might be nicer to have it be "candidate expiry date" or something otherwise it could be confusing as you might think this changes the associated pool's expiry date instead

Reply:
The mockup has "Pool expiry date" @Jerryescandon what are your thoughts?

Reply:
🤔 I see your point. Yeah I don't see why not. Makes sense to me. But would need to stay consistent with the previous one too. E.g. "Candidate pool status" + "Candidate expiry date"

Reply:
Thanks! Labels changed in [update field labels to candidate](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3957/commits/591b9a1c83b7f5fc100b1218f472aa8f74a7e4ec)